Makefile

```diff
@@ -214,3 +214,25 @@ deploy-images:
 gen-api-docs:
 	@echo ">> generating cstor 'v1' apis docs"
 	go run github.com/ahmetb/gen-crd-api-reference-docs -api-dir ../api/pkg/apis/cstor/v1 -config hack/api-docs/config.json -template-dir hack/api-docs/template -out-file docs/api-references/apis.md
+
+# If there are any external tools need to be used, they can be added by defining a EXTERNAL_TOOLS variable 
+# Bootstrap the build by downloading additional tools
+.PHONY: bootstrap
+bootstrap:
+	@for tool in  $(EXTERNAL_TOOLS) ; do \
+		echo "Installing $$tool" ; \
+		go get -u $$tool; \
+	done
+
+.PHONY: clean
+clean: 
+	@echo '--> Cleaning directory...'
+	rm -rf bin
+	rm -rf ${GOPATH}/bin/${UPGRADE}
+	rm -rf ${GOPATH}/bin/${MIGRATE}
+	rm -rf Dockerfile.upgarde
+	rm -rf Dockerfile.migrate
+	@echo '--> Done cleaning.'
+	@echo
+
+include Makefile.buildx.mk
```

Comment:
Small quirk, can we add an EOL at EOF - This is easily fixed in the IDE settings.